I/ITK/build_tarballs.jl

```diff
@@ -0,0 +1,144 @@
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ITK"
+version = v"5.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/InsightSoftwareConsortium/ITK/releases/download/v$(version)/InsightToolkit-$(version).tar.gz", "57a4471133dc8f76bde3d6eb45285c440bd40d113428884a1487472b7b71e383")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cmake -B build \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DBUILD_SHARED_LIBS:BOOL=ON \
+    -DITK_USE_SYSTEM_EXPAT:BOOL=ON \
+    -DITK_USE_SYSTEM_FFTW:BOOL=ON \
+    -DITK_USE_SYSTEM_HDF5:BOOL=ON \
+    -DITK_USE_SYSTEM_JPEG:BOOL=ON \
+    -DITK_USE_SYSTEM_TIFF:BOOL=ON \
+    -DITK_USE_SYSTEM_PNG:BOOL=ON \
+    -DITK_USE_SYSTEM_EIGEN:BOOL=ON \
+    -DITK_USE_SYSTEM_ZLIB:BOOL=ON \
+    -DITK_USE_SYSTEM_TBB:BOOL=ON \
+    -DQNANHIBIT_VALUE:BOOL=0 \
+    -DQNANHIBIT_VALUE__TRYRUN_OUTPUT:STRING=0 \
+    -DRUN_RESULT_VAR:STRING=0 \
+    -DRUN_RESULT_VAR__TRYRUN_OUTPUT:STRING=0 \
+    -DH5_LDOUBLE_TO_LONG_SPECIAL_RUN:STRING=0 \
+    -DH5_LDOUBLE_TO_LONG_SPECIAL_RUN__TRYRUN_OUTPUT:STRING=0 \
+    -DH5_LONG_TO_LDOUBLE_SPECIAL_RUN:STRING=0 \
+    -DH5_LONG_TO_LDOUBLE_SPECIAL_RUN__TRYRUN_OUTPUT:STRING=0 \
+    -DH5_LDOUBLE_TO_LLONG_ACCURATE_RUN:STRING=0 \
+    -DH5_LDOUBLE_TO_LLONG_ACCURATE_RUN__TRYRUN_OUTPUT:STRING=0 \
+    -DH5_LLONG_TO_LDOUBLE_CORRECT_RUN:STRING=0 \
+    -DH5_LLONG_TO_LDOUBLE_CORRECT_RUN__TRYRUN_OUTPUT:STRING=0 \
+    -DH5_DISABLE_SOME_LDOUBLE_CONV_RUN:STRING=0 \
+    -DH5_DISABLE_SOME_LDOUBLE_CONV_RUN__TRYRUN_OUTPUT:STRING=0 \
+    -DH5_NO_ALIGNMENT_RESTRICTIONS_RUN:STRING=0 \
+    -DH5_NO_ALIGNMENT_RESTRICTIONS_RUN__TRYRUN_OUTPUT:STRING=0 \
+    -DH5_PRINTF_LL_TEST_RUN:STRING=0 \
+    -DH5_PRINTF_LL_TEST_RUN__TRYRUN_OUTPUT:STRING=0 \
+    -DTEST_LFS_WORKS_RUN:STRING=0 \
+    -DVXL_HAS_SSE2_HARDWARE_SUPPORT:STRING=1 \
+    -DVXL_HAS_SSE2_HARDWARE_SUPPORT__TRYRUN_OUTPUT:STRING=0 \
+    -DVXL_SSE2_HARDWARE_SUPPORT_POSSIBLE:STRING=1 \
+    -DVXL_SSE2_HARDWARE_SUPPORT_POSSIBLE__TRYRUN_OUTPUT:STRING=0 \
+    -DVCL_HAS_LFS:STRING=1 \
+    -DVCL_HAS_LFS__TRYRUN_OUTPUT:STRING=0 \
+    -DDOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS:STRING=1 \
+    -DDOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS__TRYRUN_OUTPUT:STRING=0 \
+    -D_libcxx_run_result:STRING=0 \
+    -D_libcxx_run_result__TRYRUN_OUTPUT:STRING="success" \
+    -DHAVE_CLOCK_GETTIME_RUN:STRING=0 \
+    InsightToolkit-5.3.0/
+cmake --build build --parallel ${nproc}
+cmake --install build
+install_license ${WORKSPACE}/srcdir/InsightToolkit-5.3.0/LICENSE
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+
+    Platform("x86_64", "windows"; ),
+    # Platform("x86_64", "macOS"; ),
```

Comment:
Apparently, macOS have some linking issues with the library, I tried building for aarch and arm, and couldn't succeed for a while. So my mentor suggested to appeal a PR with x86_64 platforms, since the resulting wrapper is supposed to be tested on those target systems for a while.